input/haleu/inputs/scenario13.xml

```diff
@@ -1,4 +1,4 @@
-<simulation xml:base="/home/abachmann/transition-scenarios/input/haleu/inputs/" xmlns:xi="http://www.w3.org/2001/XInclude">
+<simulation xml:base="/home/abachmann@anl.gov/transition-scenarios/input/haleu/inputs/" xmlns:xi="http://www.w3.org/2001/XInclude">
```

Comment:
I guess this is necessary for your code to run. My only question is, can this be done with just a relative path? If so, please do.

Reply:
I won't go through every file that has this pattern, but know that this comment holds for every file that contains this pattern.

Reply:
I've updated these inputs to use a relative path, and added a note in the README about the directory to run these files in.